java/src/main/java/svix/Webhook.java

```diff
@@ -0,0 +1,62 @@
+package svix;
+
+import java.net.http.HttpHeaders;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+import java.util.Optional;
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+public class Webhook {
+	private static final String HMAC_SHA512 = "HmacSHA512";
+
+	private final byte[] key;
+
+	public Webhook(String secret) {
+		this.key = Base64.getDecoder().decode(secret);
+	}
+
+	public String verify(String payload, HttpHeaders headers) throws WebhookVerificationError {
+		Optional<String> msgId = headers.firstValue("svix-id");
+		Optional<String> msgSignature = headers.firstValue("svix-signature");
+		Optional<String> msgTimestamp = headers.firstValue("svix-timestamp");
+
+		if (msgId.isEmpty() || msgSignature.isEmpty() || msgTimestamp.isEmpty()) {
+			throw new WebhookVerificationError("Missing required headers");
+		}
+
+		String toSign = String.format("%s.%s.%s", msgId, msgSignature, msgTimestamp);
```

Comment:
This should be
```java
String toSign = String.format("%s.%s.%s", msgId, msgTimestamp, payload);
```

Reply:
Yeap, missed that before, fixed. thanks!